.travis.yml

```diff
@@ -51,14 +51,15 @@ jobs:
   include:
     - stage: baseline builds
```

Comment:
"builds" is redundant, everything is a build.

Reply:
Ok, renamed it.